CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -80,6 +80,7 @@ protected function getTemplateUpdates() {
           ['name' => 'event_online_receipt', 'type' => 'text'],
           ['name' => 'event_online_receipt', 'type' => 'html'],
           ['name' => 'event_online_receipt', 'type' => 'subject'],
+          ['name' => 'contribution_invoice_receipt', 'type' => 'html'],
```

Comment:
this doesn't seem right to me i think we need a new array for 5.5.alpha1 given that is the version master is at.